jest/defs.bzl

```diff
@@ -105,21 +120,33 @@ def jest_test(
 
             See https://jestjs.io/docs/configuration#testsequencer-string for more information on Jest testSequencer config option.
 
-        snapshot_ext: The expected extensions for snapshot files. Defaults to `.snap`, the Jest default.
+        snapshots_ext: The expected extensions for snapshot files. Defaults to `.snap`, the Jest default.
+
+        silent_on_success: Whether the snapshot update generator target should write to stdout & stderr on success.
 
         jest_repository: Name of the repository created with jest_repositories().
 
         **kwargs: All other args from `js_test`. See https://github.com/aspect-build/rules_js/blob/main/docs/js_binary.md#js_test
     """
     tags = kwargs.pop("tags", [])
 
+    snapshot_files = []
+    if snapshots == True:
+        snapshots = "__snapshots__"  # default jest snapshots directory
+    if type(snapshots) == "string":
```

Comment:
this alternative usage doesn't seem to be included in the docstring?

Reply:
it is but more of a footnote
```
If the name of the snapshot directory is not the default
            `__snapshots__` because of a custom snapshot resolver, you can specify a custom directory name by setting
            `snapshots` to the custom directory name instead of True.
```

it should a pretty rare case. 99.9% of the time project will use the default `__snapshots__` directory.

Reply:
oh I see it at the end of the second paragraph, okay!